static/public.css

```diff
@@ -4548,6 +4548,11 @@ img + #detail-art-text {
     color: #939699;
 }
 
+#footer-info a,
+#footer-info a:visited {
```

Comment:
Whoops, apparently not. I added it when trying to figure out why the color wasn't changing, turned out that for some reason Safari wasn't reloading the stylesheet. Removed now.